### PR TITLE
SPARC community terms sync for 2022-07-10T2324Z

### PR DESCRIPTION
**Pull request by tgbugs**
synchronize latest changes via http

### CODE REVIEW

File: ttl/sparc-community-terms.ttl [hunk 1]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    rdfs:label "proximoventral midpoint on serosa of upper esophageal sphincter" ;
+    definition: "The point on the serosa of the esophagus where the most proximal  (oral) edge of the upper esophageal sphincter coincides with the ventral midpoint of the esophagus." ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793182 ;
+    ilxtr:hasIlxId ILX:0793182 ;
+    ilxtr:hasIlxPreferredId ILX:0793182 ;
+    ilxtr:includedForSPARCUnder FMA:7131 .
+
+ILX:0793207 a owl:Class ;
+    rdfs:label "Ventral root of the thirteenth thoracic nerve " ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:6851]: trailing whitespace

File: ttl/sparc-community-terms.ttl [hunk 2]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    rdfs:label "T11 - T12 Interganglionic segment" ;
+    definition: "Nerve fibers connecting the eleventh and twelfth thoracic sympathetic ganglia in the sympathetic trunk." ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793240 ;
+    ilxtr:hasIlxId ILX:0793240 ;
+    ilxtr:hasIlxPreferredId ILX:0793240 .
+
+ILX:0793242 a owl:Class ;
+    rdfs:label "T12 - L1 Interganglionic segment" ;
+    definition: "Nerve fibers connecting the twelfth thoracic sympathetic ganglia and the first lumbar sympathetic ganglia in the sympathetic trunk. " ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7070]: trailing whitespace

File: ttl/sparc-community-terms.ttl [hunk 3]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ILX:0739239 "True" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793547 ;
+    ilxtr:hasIlxId ILX:0793547 ;
+    ilxtr:hasIlxPreferredId ILX:0793547 ;
+    ilxtr:includedForSPARCUnder FMA:7088 .
+
+ILX:0793548 a owl:Class ;
+    rdfs:label "outer surface of epicardium" ;
+    definition: "The outermost 2-D surface of the heart where it meets the pericardial cavity, outside surface of all solid tissue in wall of heart. " ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7596]: trailing whitespace

File: ttl/sparc-community-terms.ttl [hunk 4]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ILX:0383242 FMA:9461 ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793549 ;
+    ilxtr:hasIlxId ILX:0793549 ;
+    ilxtr:hasIlxPreferredId ILX:0793549 .
+
+ILX:0793551 a owl:Class ;
+    rdfs:label "Gray communicating ramus of the first cervical nerve" ;
+    NIFRID:synonym "C1 gray ramus" ;
+    definition: "The postganglionic nerve fibers from superior cervical ganglion passing to the first cervical spinal nerve. " ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7619]: trailing whitespace

File: ttl/sparc-community-terms.ttl [hunk 5]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+ILX:0793558 a owl:Class ;
+    rdfs:label "distal stomach" ;
+    definition: "The section of the stomach comprising the antrum and pylorus." ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793558 ;
+    ilxtr:hasIlxId ILX:0793558 ;
+    ilxtr:hasIlxPreferredId ILX:0793558 .
+
+ILX:0793559 a owl:Class ;
+    rdfs:label "bladder nerve " ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7677]: trailing whitespace

File: ttl/sparc-community-terms.ttl [hunk 6]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+
+ILX:0793558 a owl:Class ;
+    rdfs:label "distal stomach" ;
+    definition: "The section of the stomach comprising the antrum and pylorus." ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793558 ;
+    ilxtr:hasIlxId ILX:0793558 ;
+    ilxtr:hasIlxPreferredId ILX:0793558 .
+
+ILX:0793559 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7676]: missing definition

File: ttl/sparc-community-terms.ttl [hunk 7]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+
+ILX:0793575 a owl:Class ;
+    rdfs:label "blood vessel of pancreas" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793575 ;
+    ilxtr:hasIlxId ILX:0793575 ;
+    ilxtr:hasIlxPreferredId ILX:0793575 .
+
+ILX:0793615 a owl:Class ;
+    rdfs:label "Ventral root of the sixth lumbar nerve " ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7748]: trailing whitespace

File: ttl/sparc-community-terms.ttl [hunk 8]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+
+ILX:0793628 a owl:Class ;
+    rdfs:label "Platelet-Derived Growth Factor Receptor Alpha" ;
+    definition: "Platelet-derived growth factor receptor alpha binds to certain isoforms of platelet-derived growth factors (PDGFs) and thereby becomes active in stimulating cell signaling pathways that elicit responses such as cellular growth and differentiation." ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793628 ;
+    ilxtr:hasIlxId ILX:0793628 ;
+    ilxtr:hasIlxPreferredId ILX:0793628 .
+
+ILX:0793629 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7828]: why is this class being differentiated?

File: ttl/sparc-community-terms.ttl [hunk 9]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+ILX:0793615 a owl:Class ;
+    rdfs:label "Ventral root of the sixth lumbar nerve " ;
+    NIFRID:synonym "Anterior root of sixth lumbar nerve" ;
+    definition: "The motor axons associated with the sixth lumbar spinal segment." ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793615 ;
+    ilxtr:hasIlxId ILX:0793615 ;
+    ilxtr:hasIlxPreferredId ILX:0793615 .
+
+ILX:0793616 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7756]: missing def

File: ttl/sparc-community-terms.ttl [hunk 10]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ilxtr:hasIlxPreferredId ILX:0793573 .
+
+ILX:0793574 a owl:Class ;
+    rdfs:label "intracranial nerves" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793574 ;
+    ilxtr:hasIlxId ILX:0793574 ;
+    ilxtr:hasIlxPreferredId ILX:0793574 .
+
+ILX:0793575 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7740]: why is this being differentiated

missing def

File: ttl/sparc-community-terms.ttl [hunk 11]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ilxtr:hasIlxPreferredId ILX:0793572 .
+
+ILX:0793573 a owl:Class ;
+    rdfs:label "terminal bronchiole parasympathetic ganglia" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793573 ;
+    ilxtr:hasIlxId ILX:0793573 ;
+    ilxtr:hasIlxPreferredId ILX:0793573 .
+
+ILX:0793574 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7733]: missing definition

File: ttl/sparc-community-terms.ttl [hunk 12]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ilxtr:hasIlxPreferredId ILX:0793571 .
+
+ILX:0793572 a owl:Class ;
+    rdfs:label "bronchiole parasympathetic ganglia" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793572 ;
+    ilxtr:hasIlxId ILX:0793572 ;
+    ilxtr:hasIlxPreferredId ILX:0793572 .
+
+ILX:0793573 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7726]: missing def

File: ttl/sparc-community-terms.ttl [hunk 13]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ilxtr:hasIlxPreferredId ILX:0793570 .
+
+ILX:0793571 a owl:Class ;
+    rdfs:label "bronchus parasympathetic ganglia" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793571 ;
+    ilxtr:hasIlxId ILX:0793571 ;
+    ilxtr:hasIlxPreferredId ILX:0793571 .
+
+ILX:0793572 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7719]: missing def

File: ttl/sparc-community-terms.ttl [hunk 14]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ilxtr:hasIlxPreferredId ILX:0793564 .
+
+ILX:0793570 a owl:Class ;
+    rdfs:label "trachea parasympathetic ganglia" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793570 ;
+    ilxtr:hasIlxId ILX:0793570 ;
+    ilxtr:hasIlxPreferredId ILX:0793570 .
+
+ILX:0793571 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7712]: missing def

File: ttl/sparc-community-terms.ttl [hunk 15]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ilxtr:hasIlxPreferredId ILX:0793563 .
+
+ILX:0793564 a owl:Class ;
+    rdfs:label "oblique muscle layer of esophagus" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793564 ;
+    ilxtr:hasIlxId ILX:0793564 ;
+    ilxtr:hasIlxPreferredId ILX:0793564 .
+
+ILX:0793570 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7705]: missing def

File: ttl/sparc-community-terms.ttl [hunk 16]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+    ilxtr:hasIlxPreferredId ILX:0793562 .
+
+ILX:0793563 a owl:Class ;
+    rdfs:label "splenic nerve" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793563 ;
+    ilxtr:hasIlxId ILX:0793563 ;
+    ilxtr:hasIlxPreferredId ILX:0793563 .
+
+ILX:0793564 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7698]: missing def

File: ttl/sparc-community-terms.ttl [hunk 17]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+
+ILX:0793562 a owl:Class ;
+    rdfs:label "internal branch of the inferior laryngeal branch of the vagus nerve" ;
+    definition: "there's a left and right recurrent laryngeal nerve, but not inferior in FMA" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793562 ;
+    ilxtr:hasIlxId ILX:0793562 ;
+    ilxtr:hasIlxPreferredId ILX:0793562 .
+
+ILX:0793563 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7691]: missing def

File: ttl/sparc-community-terms.ttl [hunk 18]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+ILX:0793559 a owl:Class ;
+    rdfs:label "bladder nerve " ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793559 ;
+    ilxtr:hasIlxId ILX:0793559 ;
+    ilxtr:hasIlxPreferredId ILX:0793559 .
+
+ILX:0793562 a owl:Class ;
+    rdfs:label "internal branch of the inferior laryngeal branch of the vagus nerve" ;
+    definition: "there's a left and right recurrent laryngeal nerve, but not inferior in FMA" ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7685]: bad english

File: ttl/sparc-community-terms.ttl [hunk 19]
```diff
@@ -6466,6 +6811,1028 @@ ILX:0793138 a owl:Class ;
...
+ILX:0793556 a owl:Class ;
+    rdfs:label "ventricular intrinsic cervical ganglion" ;
+    NIFRID:synonym "Ventricular ganglionated plexus" ;
+    definition: "Autonomic ganglia residing on the epicardial surface of the ventricles.  Classically regarded as parasympathetic postganglionic neurons, but may include cholinergic, adrenergic, and/or nitrergic cells receiving, principally, cholinergic innervation from the preganglionic parasympathetic neurons of the brainstem." ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId ILX:0793556 ;
+    ilxtr:hasIlxId ILX:0793556 ;
+    ilxtr:hasIlxPreferredId ILX:0793556 .
+
+ILX:0793557 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:7661]: missing def

File: ttl/sparc-community-terms.ttl [hunk 20]
```diff
@@ -5331,6 +5647,30 @@ ILX:0743583 a owl:Class ;
     ilxtr:hasIlxId ILX:0743583 ;
     ilxtr:hasIlxPreferredId ILX:0743583 .
 
+ILX:0773136 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:5650]: missing def

File: ttl/sparc-community-terms.ttl [hunk 21]
```diff
@@ -5331,6 +5647,30 @@ ILX:0743583 a owl:Class ;
...
+    rdfs:label "Myocardium of right atrium" ;
+    ilxtr:hasExistingId ILX:0773136 ;
+    ilxtr:hasIlxId ILX:0773136 ;
+    ilxtr:hasIlxPreferredId ILX:0773136 .
+
+ILX:0774799 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:5656]: missing def

File: ttl/sparc-community-terms.ttl [hunk 22]
```diff
@@ -5331,6 +5647,30 @@ ILX:0743583 a owl:Class ;
...
+    rdfs:label "Myocardium of left ventricle" ;
+    ilxtr:hasExistingId ILX:0774799 ;
+    ilxtr:hasIlxId ILX:0774799 ;
+    ilxtr:hasIlxPreferredId ILX:0774799 .
+
+ILX:0774965 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:5662]: missing def

File: ttl/sparc-community-terms.ttl [hunk 23]
```diff
@@ -5331,6 +5647,30 @@ ILX:0743583 a owl:Class ;
...
+    rdfs:label "Myocardium of right ventricle" ;
+    ilxtr:hasExistingId ILX:0774965 ;
+    ilxtr:hasIlxId ILX:0774965 ;
+    ilxtr:hasIlxPreferredId ILX:0774965 .
+
+ILX:0775551 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:5668]: missing def

File: ttl/sparc-community-terms.ttl [hunk 24]
```diff
@@ -2869,10 +3169,19 @@ FMA:68871 a owl:Class ;
         "substantia gelationsa centralis",
         "substantia gliosa (spinalis)" .
 
+FMA:69262 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:3172]: missing def

needs existing id to CL

File: ttl/sparc-community-terms.ttl [hunk 25]
```diff
@@ -2402,21 +2625,32 @@ FMA:62004 a owl:Class ;
...
-    ilxtr:hasIlxId ILX:0106736 ;
+    ilxtr:hasIlxId ILX:0106736,
+        ILX:0738301 ;
     ilxtr:hasIlxPreferredId UBERON:0001896 ;
     oboInOwl:hasBroadSynonym "medulla" ;
     oboInOwl:hasExactSynonym "bulbus" ;
     oboInOwl:hasRelatedSynonym "medulla oblonzata",
         "metepencephalon" .
 
+FMA:62122 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:2641]: missing def

needs existing id to CL

File: ttl/sparc-community-terms.ttl [hunk 26]
```diff
@@ -576,6 +584,26 @@ FMA:6171 a owl:Class ;
     ilxtr:hasIlxId ILX:0739246 ;
     ilxtr:hasIlxPreferredId FMA:6171 .
 
+FMA:6240 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:587]: missing def

File: ttl/sparc-community-terms.ttl [hunk 27]
```diff
@@ -576,6 +584,26 @@ FMA:6171 a owl:Class ;
...
+    rdfs:label "Internal laryngeal nerve" ;
+    NIFRID:synonym "Internal laryngeal nerve" ;
+    ilxr:type ilx.type:term ;
+    ilxtr:hasExistingId FMA:6240,
+        ILX:0793561 ;
+    ilxtr:hasExternalId FMA:6240 ;
+    ilxtr:hasIlxId ILX:0793561 ;
+    ilxtr:hasIlxPreferredId FMA:6240 .
+
+FMA:6243 a owl:Class ;
```
Comment (tgbugs) [inline ttl/sparc-community-terms.ttl:597]: missing def

**Review by tgbugs — COMMENTED** [review]
@MCSZ @stappan there are a number of issues that we need to address, I can merge this, but we need these to be fixed.

**Comment by tgbugs** [comment]
While we're here, it is also worth having a search for malformed DOIs in https://github.com/SciCrunch/NIF-Ontology/blob/961588609a80fc1b384d5cba6adde13df85587f2/ttl/sparc-community-terms.ttl.

`Ctrl-f` `&lt` will reveal a number of cases where bad encoding is present.

File: ttl/sparc-community-terms.ttl (hunk 5 above)
Comment (tgbugs) [reply ttl/sparc-community-terms.ttl:7677]: I went ahead an fixed this one.

File: ttl/sparc-community-terms.ttl (hunk 4 above)
Comment (MCSZ) [reply ttl/sparc-community-terms.ttl:7619]: removed whitespace http://uri.interlex.org/base/ilx_0793551

File: ttl/sparc-community-terms.ttl (hunk 3 above)
Comment (MCSZ) [reply ttl/sparc-community-terms.ttl:7596]: whitespace removed http://uri.interlex.org/base/ilx_0793548

File: ttl/sparc-community-terms.ttl (hunk 1 above)
Comment (MCSZ) [reply ttl/sparc-community-terms.ttl:6851]: whitespace removed http://uri.interlex.org/base/ilx_0793207

File: ttl/sparc-community-terms.ttl (hunk 6 above)
Comment (MCSZ) [reply ttl/sparc-community-terms.ttl:7676]: removed whitespace http://uri.interlex.org/base/ilx_0793615

File: ttl/sparc-community-terms.ttl (hunk 7 above)
Comment (MCSZ) [reply ttl/sparc-community-terms.ttl:7748]: removed whitespace http://uri.interlex.org/base/ilx_0793615

**Pull request merged**